k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -965,6 +968,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.read_receipt:
                 onReadReceipt();
                 break;
+            case R.id.is_urgent:
+                if (!isHighPriority) {
+                    item.setTitle("Normal");
+                    this.isHighPriority = true;
+                    setTitle(getTitle() + ":U");
+                } else {
+                    item.setTitle("Urgent");
+                    this.isHighPriority = false;
+                    setTitle(getTitle().toString().split(":")[0]);
```

Comment:
this seems very volatile. I'm also not sure if the ":U" is necessary, it's not very obvious what it means and probably has little informational value to the user.